modules/swagger-codegen/src/main/resources/Java/libraries/retrofit2/pom.mustache

```diff
@@ -213,6 +213,18 @@
       <version>${retrofit-version}</version>
     </dependency>
     {{/useRxJava}}
+    {{#useRxJava2}}
+    <dependency>
+      <groupId>io.reactivex.rxjava2</groupId>
+      <artifactId>rxjava</artifactId>
+      <version>${rxjava-version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.jakewharton.retrofit</groupId>
+      <artifactId>com.jakewharton.retrofit</artifactId>
```

Comment:
I think this should be `<artifactId>retrofit2-rxjava2-adapter</artifactId>` instead.